src/Behavioral.Automation/Bindings/DropdownBinding.cs

```diff
@@ -77,6 +87,39 @@ public void CheckMultipleSelectedValues([NotNull] IMultiSelectDropdownWrapper wr
             wrapper.SelectedValuesTexts.Should().BeEquivalentTo(values.Rows.Select(x => x.Values.Single()));
         }
 
+        [Given("the \"(.*?)\" value (is|become) (enabled|disabled) in (.*?)")]
```

Comment:
Could you please update the Bindings.md?

Reply:
Updated